doc/holochain_101/src/welcome.md

```diff
@@ -1 +1,9 @@
 # Welcome
+
+Welcome, you've discovered the book on Holochain. It will guide you comprehensively through everything you need to know about the creation of fully peer-to-peer, secure, and decentralized applications built with Holochain. It also includes a section on contributing to, and extending, Holochain itself.
+
+Holochain is an open source software library that provides a comprehensive solution for businesses, communities, and any group of people to build and run applications which are hosted and validated by the "users" themselves, providing a superior level of agency and autonomy over heavy reliance on the so-called "cloud" and third parties. "Users" cease to merely "use" an application, and by running it, participate equally as nodes in the network, hosting and validating data, in addition to consuming it. Holochain provides a cross-platform framework for the development and execution of these applications. Applications can be developed utilizing any of the major operating systems in use, and ran on any device with a 64-bit architecture, from desktops to laptops to Android (arm64) devices, opening a whole new set of opportunities for the p2p applications you've been dreaming and scheming about. 
```

Comment:
> "Users" cease to merely "use" an application, and by running it, participate equally as nodes in the network, hosting and validating data, in addition to consuming it. 
replace with: "Users" cease to merely "use" an application.  By running it, they participate equally as nodes in the network, hosting and validating data, in addition to consuming it.

Reply:
"Applications can be developed utilizing any of the major operating systems in use, and ran on any device with a 64-bit architecture, from desktops to laptops to Android (arm64) devices, opening a whole new set of opportunities for the p2p applications you've been dreaming and scheming about. "
replace with: 
"Applications can be developed utilizing any of the major operating systems in use, and run on any device with a 64-bit architecture.  From desktops to laptops to Android (arm64) devices, a whole new set of opportunites open for the p2p applications you've been dreaming and scheming about."